services/director-v2/setup.py

```diff
@@ -3,56 +3,64 @@
-    raise RuntimeError(
-        "Expected ~=3.8, got %s (Tip: did you forget to 'source .venv/bin/activate' or 'pyenv local'?)"
-        % str(sys.version_info)
-    )
-
-current_dir = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve().parent
+def read_reqs(reqs_path: Path) -> Set[str]:
+    return {
+        r
+        for r in re.findall(
+            r"(^[^#\n-][\w\[,\]]+[-~>=<.\w]*)",
+            reqs_path.read_text(),
+            re.MULTILINE,
+        )
+        if isinstance(r, str)
+    }
 
 
-def read_reqs(reqs_path: Path):
-    return re.findall(
-        r"(^[^#\n-][\w\[,\]]+[-~>=<.\w]*)", reqs_path.read_text(), re.MULTILINE
-    )
+CURRENT_DIR = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve().parent
 
+NAME = "simcore-service-director-v2"
+VERSION = (CURRENT_DIR / "VERSION").read_text().strip()
+AUTHORS = "Sylvain Anderegg (sanderegg), Pedro Crespo (pcrespov)"
+README = (CURRENT_DIR / "README.md").read_text()
 
-readme = (current_dir / "README.md").read_text()
-version = (current_dir / "VERSION").read_text().strip()
-
-install_requirements = read_reqs(current_dir / "requirements" / "_base.txt") + [
-    "simcore-models-library",
-    "simcore-settings-library",
-    "simcore-postgres-database",
-]
-
-test_requirements = read_reqs(current_dir / "requirements" / "_test.txt")
-
-
-setup(
-    name="simcore-service-director-v2",
-    version=version,
-    author="Sylvain Anderegg (sanderegg), Pedro Crespo (pcrespov)",
-    description="Orchestrates the pipeline of services defined by the user",
-    classifiers=[
-        "Development Status :: 1 - Planning",
-        "License :: OSI Approved :: MIT License",
-        "Natural Language :: English",
-        "Programming Language :: Python :: 3.8",
-    ],
-    long_description=readme,
-    license="MIT license",
-    python_requires="~=3.8",
-    packages=find_packages(where="src"),
-    package_dir={
-        "": "src",
-    },
-    install_requires=install_requirements,
-    test_suite="tests",
-    tests_require=test_requirements,
-    extras_require={"test": test_requirements},
+PROD_REQUIREMENTS = tuple(
+    read_reqs(CURRENT_DIR / "requirements" / "_base.txt")
+    | {
+        "simcore-models-library",
+        "simcore-postgres-database",
+        "simcore-service-library[fastapi]",
+        "simcore-settings-library",
+    }
 )
+
+TEST_REQUIREMENTS = tuple(read_reqs(CURRENT_DIR / "requirements" / "_test.txt"))
+
+
+if __name__ == "__main__":
+    from setuptools import find_packages, setup
+
+    setup(
+        name=NAME,
+        version=VERSION,
+        author=AUTHORS,
+        description="Orchestrates the pipeline of services defined by the user",
+        classifiers=[
+            "Development Status :: 1 - Planning",
+            "License :: OSI Approved :: MIT License",
+            "Natural Language :: English",
+            "Programming Language :: Python :: 3.8",
+        ],
```

Comment:
I see we still kept classifiers here. Also the description was not being moved above.